var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -255,18 +264,28 @@ def patch(self):
                         '-I{0}'.format(spec['zlib'].prefix.include),
                         'gcc/Makefile.in')
 
+    # https://gcc.gnu.org/install/configure.html
     def configure_args(self):
         spec = self.spec
 
         # Generic options to compile GCC
         options = [
+            # Distributor options
+            '--with-pkgversion=Spack GCC',
+            '--with-bugurl=https://github.com/spack/spack/issues',
+            # Xcode 10 dropped 32-bit support
             '--disable-multilib',
+            # Don't bother with full 3-stage bootstrapping process
+            # Builds much faster, but doesn't test build as well
+            # See https://gcc.gnu.org/install/build.html
+            '--disable-bootstrap',
             '--enable-languages={0}'.format(
                 ','.join(spec.variants['languages'].value)),
+            # Drop gettext and libiconv dependencies
+            '--disable-nls',
+            '--with-libiconv-prefix={0}'.format(spec['libiconv'].prefix),
             '--with-mpfr={0}'.format(spec['mpfr'].prefix),
             '--with-gmp={0}'.format(spec['gmp'].prefix),
-            '--enable-lto',
```

Comment:
This flag is already enabled by default. I can add it back if you want.

Reply:
nah keep it default.  It might be nice to have as a variant but nobody is currently asking for it so enabled sounds good.

Reply:
I think I copied some of these options from another build script. It is possible those flags are version dependent. I have no objection to removing them but you should check if the Homebrew formulas for older gcc versions had them.